### PR TITLE
Add update_test_reference target and cleanup old target

### PR DESCRIPTION
**Pull request by ericvn**
update_examples target removed in https://github.com/istio/istio.io/pull/6105

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: Makefile.core.mk [hunk 1]
```diff
@@ -140,7 +140,10 @@ netlify: netlify_install
 update_ref_docs:
 	@scripts/grab_reference_docs.sh $(SOURCE_BRANCH_NAME)
 
-update_all: update_ref_docs update_examples
```
Comment (davidhauck) [inline Makefile.core.mk:143]: Do we not need update_examples any more?

Comment (ericvn) [reply Makefile.core.mk:143]: There is no target, it was removed in an earlier PR (noted above). And I removed the script called in that target.

**Review by davidhauck — APPROVED** [review]

**Pull request merged**